Makefile

```diff
@@ -8,6 +8,7 @@ all:
 clean:
 	find . -name "*.so" -o -name "*.pyc" -o -name "*.md5" -o -name "*.pyd" | xargs rm -f
 	find . -name "*.pyx" -exec ./tools/rm_pyx_c_file.sh {} \;
+	rm -f MANIFEST
```

Comment:
Where do you get this file from?

Reply:
When you do source builds, a `MANIFEST` file is generated from `MANIFEST.in`.

Reply:
It is actually created during `python setup.py sdist` run.

Reply:
How is that not a source build? :)

Reply:
For me, `python setup.py build_ext` or `pip install -e .` are source builds. `sdist` is `sdist`. :) Anyways...